scene/resources/visual_shader_nodes.h

```diff
@@ -199,7 +199,8 @@ class VisualShaderNodeTexture : public VisualShaderNode {
 		SOURCE_SCREEN,
 		SOURCE_2D_TEXTURE,
 		SOURCE_2D_NORMAL,
-		SOURCE_DEPTH
+		SOURCE_DEPTH,
+		SOURCE_PORT
```

Comment:
Add trailing comma to avoid having a diff on the last entry when you add a new one next time.